_i18n/es/resources/user-guides/prove-payment.md

```diff
@@ -1,86 +1,78 @@
 {% assign version = '1.1.0' | split: '.' %}
-{% include disclaimer.html translated="false" version=page.version %}
-### Prove payments
+{% include disclaimer.html translated="true" version=page.version %}
+### Probar pagos
 
-When you send money to a party who then disputes the payment was made, you need to be able to prove the payment was made.
+Cuando envías dinero a un segundo quien después reclama que el pago haya sido realizado, necesitas ser capaz de probar que el pago fue realizado.
 
-With Bitcoin, this is typically done by looking up the transaction ID, where the origin and destination addresses are
-shown, along with the amount transacted.
+Con Bitcoin, esto es típicamente hecho observando el ID de transacción, donde se muestran las direcciones de origen y destino, junto con la cantidad tramitada.
 
-Monero, however, is private: that information is not available publicly on the blockchain. The steps are therefore a bit
-more involved.
+Monero, no obstante, es privado: esa información no está disponible públicamente en la blockchain. Los pasos son, por lo tanto, un poco más complejos.
 
-To prove to Charlie that she made a payment to Bob, Alice must supply Charlie three pieces of information:
+Para probarle a Charlie que ella hizo un pago a Bob, Alicia debe proveer a Charlie con tres piezas de información:
 
-- the transaction ID, as is done in Bitcoin
-- Bob's address, as is done with Bitcoin
-- the transaction's key, which is new with Monero and other CryptoNote currencies
+- el ID de transacción, como es hecho en Bitcoin
+- la dirección de Bob, como es hecho en Bitcoin
+- la llave de transacción, que es algo nuevo con Monero y otras criptomonedas CryptoNote
 
-When Alice made the transaction, a one time key was automatically generated just for this transaction.
+Cuando Alicia hizo la transacción, una llave de un solo uso fue automáticamente generada para esa transacción.
 
 #### CLI
 
-Alice can query it thus in monero-wallet-cli (new name for the old simplewallet):
+Alicia puede consultarlo así en monero-wallet-cli (nuevo nombre para la vieja simplewallet):
 
 > get_tx_key TXID
 
-Alice would plug in her actual transaction ID instead of this TXID placeholder. All being well, the one time transaction key
-will be displayed.
+Alicia pondría su actual ID de transacción en lugar del marcador TXID. Si todo va bien, la llave de un solo uso será mostrada.
 
-Note that this will only work if monero-wallet-cli is set to save transaction keys. To double check:
+Tenga en cuenta que esto sólo funcionará si monero-wallet-cli está configurado para guardar llaves de transacción. Para revisar doblemente:
```

Comment:
"Para revisar doblemente:" sounds really strange. 

I suggest:
"Para volver a revisar:"

Reply:
Double-check is an expression:

https://www.linguee.com/english-spanish/search?source=auto&query=double-check&cw=358

Reply:
"Para volver a comprobar:" :smile: haha